src/main/java/cascading/tap/hive/HivePartitionTap.java

```diff
@@ -123,6 +160,91 @@ public void sourceConfInit( FlowProcess<? extends Configuration> flowProcess, Co
     super.sourceConfInit( flowProcess, conf );
     }
 
+
+  @Override
+  public String[] getChildPartitionIdentifiers( FlowProcess<? extends Configuration> flowProcess, boolean fullyQualified ) throws IOException
+    {
+    Configuration conf = flowProcess.getConfig();
+    IMetaStoreClient metaStoreClient = null;
+    try
+      {
```

Comment:
which hive versions was this tested with? We try to support older releases by not using APIs, that are only present in later releases. I just would like to understand the impact.